mattermost-plugin/webapp/src/index.tsx

```diff
@@ -87,12 +87,12 @@ function customHistory() {
             }
 
             const pathName = event.data.message?.pathName
-            if (!pathName || !pathName.startsWith(windowAny.frontendBaseURL)) {
```

Comment:
This solution would break things like having focalboard in a subpath

Reply:
@jespino this is needed to make subpaths work in the desktop app. This code doesn't affect the ability to run Focalboard with subpath in browsers.